lang/java/pom.xml

```diff
@@ -65,6 +65,7 @@
     <exec-plugin.version>3.0.0</exec-plugin.version>
     <file-management.version>3.0.0</file-management.version>
     <javacc-plugin.version>2.6</javacc-plugin.version>
+    <mockito.version>3.9.0</mockito.version>
```

Comment:
Why 3.9.0 ?
Current latest is 3.12.4

Reply:
That's because 3.9.0 is the most widly used/stable version of Mockito

Reply:
Do you say that there are some regressions in the newer versions ?

Reply:
Not sure if there are regressions in the newer version, but according to [Maven Repository](https://mvnrepository.com/artifact/org.mockito/mockito-core), 3.9.0 is the most well accepted version.

Reply:
3.0.0 has even more usages than 3.9.0 :-)
I hope you understand that this metric should not lead you.

Reply:
I see, thanks for the advice!